python/cli/diagnose/diagnose.py

```diff
@@ -122,7 +122,7 @@ def addMeasurement(type, t, v):
                     if measurementType == "gyroscope":
                         latestGyroTime = t
                         for f in framesMissingNextGyroTime:
-                            gyroTimeDeltas["next"] = abs(latestGyroTime - gyroTimeDeltas["t"])
+                            f["next"] = abs(latestGyroTime - f["t"])
```

Comment:
Bug was that `gyroTimeDeltas` doesn't refer to f when there are multiple frames on one line.